src/Vixen.Common/AudioPlayer/FileReader/AudioFileReader.cs

```diff
@@ -20,20 +21,28 @@ public class AudioFileReader : WaveStream, ISampleProvider
         private readonly int _sourceBytesPerSample;
         private readonly long _length;
         private readonly object _lockObject;
+		private static readonly Logger Logging = LogManager.GetCurrentClassLogger();
 
-        /// <summary>
-        /// Initializes a new instance of AudioFileReader
-        /// </summary>
-        /// <param name="fileName">The file to open</param>
-        public AudioFileReader(string fileName)
+		/// <summary>

```

Comment:
Looks like the indentation changed here by accident.

Reply:
There are definitely places this can happen. I try to revert them back. Just for reference, I am ok with fixing them in their own commit separate from actual code changes when they are not part of the actual code change.